addons/mrp/views/mrp_workcenter_views.xml

```diff
@@ -499,7 +499,7 @@
         <field name="model">mrp.workcenter.productivity</field>
         <field name="arch" type="xml">
             <search string="Operations">
-                <field name="workcenter_id"/>
+                <field name="production_id"/>
```

Comment:
Not wrong at all, if you want to add this field, create a custom module instead :v
And v14 is no longer support by odoo

Reply:
What do u thing with the picture?
![image](https://github.com/user-attachments/assets/ee623923-b053-490d-b701-594abf97af19)

Reply:
@duybq86 Seem like they remove it in https://github.com/odoo/odoo/pull/63500 >=15.0
but still v14 is no longer support so i don't think they will merge this

Reply:
odoo is community, and i think it good for some people, they can't upgrade to new version.

Reply:
I know, good luck bro